database/scripts/lib/buildfarm_tools.rb

```diff
@@ -115,6 +115,7 @@ def self.jobs_last_success_date(older_than_days: 0)
 
```

Comment:
Do you need to change the database with this PR? :eyes:

Reply:
Yes! Is already changed 👀. But as it's a binary file you can't see the changes

Reply:
I mean, you can't see the diff. But you can see the change haha